### PR TITLE
Correct stem alignment for grace note bends on TAB

### PR DESCRIPTION
**Pull request by mike-spa**
Resolves: #20206

### CODE REVIEW

**Review by oktophonie — APPROVED** [review]

File: src/engraving/dom/chord.cpp [hunk 1]
```diff
@@ -2601,6 +2609,22 @@ bool Chord::isPreBendOrGraceBendStart() const
     return false;
 }
 
+bool Chord::isGraceBendEnd() const
```
Comment (RomanPudashkin) [inline src/engraving/dom/chord.cpp:2612]: It could probably be optimized a bit: 

```
if (isGrace() || m_graceNotes.empty()) {
    return false;
}
```

Comment (mike-spa) [reply src/engraving/dom/chord.cpp:2612]: good point, done! 👍

**Review by RomanPudashkin — APPROVED** [review]

File: src/engraving/dom/chord.cpp (hunk 1 above)
Comment (Mellobrainbox) [reply src/engraving/dom/chord.cpp:2612]: Thanks

**Review by Mellobrainbox — APPROVED** [review]
Good 👍

**Pull request merged**